package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@babbel/eslint-config",
+  "version": "1.0.0",
+  "author": "Eric L. Goldstein <egoldstein@babbel.com>",
+  "description": "Hierarchical ESLint configuration collection that intends to be simple to use, layered, and shared with others",
+  "keywords": [
+    "eslint",
+    "eslintconfig"
+  ],
+  "engines": {
+    "node": ">=16.13.0"
+  },
+  "packageManager": "npm@10.2.3",
+  "exports": {
+    ".": "./lib/eslintBaseConfig.json",
+    "./browser": "./lib/eslintBrowserConfig.json",
+    "./isomorphic": "./lib/eslintIsomorphicConfig.json",
+    "./jest": "./lib/eslintJestConfig.json",
+    "./node": "./lib/eslintNodeConfig.json",
+    "./playwright": "./lib/eslintPlaywrightConfig.json",
+    "./preact": "./lib/eslintPreactConfig.json",
+    "./preact-typescript": "./lib/eslintPreactTypescriptConfig.json",
+    "./react": "./lib/eslintReactConfig.json",
+    "./react-typescript": "./lib/eslintReactTypescriptConfig.json",
+    "./typescript": "./lib/eslintTypescriptConfig.json"
+  },
+  "files": [
+    "lib/"
+  ],
+  "scripts": {
+    "delete:node_modules": "rm -rf node_modules",
+    "delete:package-lock": "rm -f package-lock.json",
+    "format:code": "prettier --write --no-editorconfig .",
+    "list:eslint:disable-directives": "rg '/(/|\\*+)[ \\t]*eslint-disable[^*]*(\\*+/)?'",
+    "list:todo-comments": "rg --only-matching '(TODO|FIXME):[a-zA-Z0-9\\t .,;?]+'",
+    "reinstall": "npm-run-all --silent delete:node_modules delete:package-lock && npm i",
+    "reinstall:use-lock-file": "npm run --silent delete:node_modules && npm ci",
+    "validate:formatting": "prettier --check --no-editorconfig .",
+    "validate:linting:eslint": "eslint-config-prettier ./lib/eslintBaseConfig.json"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "6.7.0",
+    "@typescript-eslint/parser": "6.7.0",
+    "eslint-config-airbnb": "19.0.4",
+    "eslint-config-airbnb-base": "15.0.0",
+    "eslint-config-airbnb-typescript": "17.1.0",
+    "eslint-config-prettier": "9.0.0",
+    "eslint-plugin-compat": "4.2.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "eslint-plugin-import": "2.28.1",
+    "eslint-plugin-jest": "27.2.3",
+    "eslint-plugin-jest-dom": "5.1.0",
+    "eslint-plugin-jest-formatting": "3.1.0",
+    "eslint-plugin-jsx-a11y": "6.7.1",
+    "eslint-plugin-n": "16.1.0",
+    "eslint-plugin-playwright": "0.16.0",
+    "eslint-plugin-regexp": "1.15.0",
+    "eslint-plugin-security": "1.7.1",
+    "eslint-plugin-unicorn": "48.0.1"
```

Comment:
All dependencies are identical relative to the latest version of `@lessonnine/eslint-config-usa-tech.js` in case there are any problems after publishing this open source. When the open source version is confirmed published and working, I'll update all dependencies to latest.